src/theme.css

```diff
@@ -135,25 +135,30 @@ body {
 }
 
 :root {
-  --background-catalyst: #4888EE;
-  --background-poi: #44B600;
-  --background-ban-name: #FF5AFE;
-  --background-grant: #3C24B3;
-  --background-poll: #FF7439;
-  --background-remove-poi: #FF247C;
-  --background-happy-green: #44B600;
-  --background-linked-wearables: #FFBC5B;
+  --background-catalyst: #4888ee;
+  --background-poi: #44b600;
+  --background-ban-name: #ff5afe;
+  --background-grant: #3c24b3;
+  --background-poll: #ff7439;
+  --background-remove-poi: #ff247c;
+  --background-happy-green: #44b600;
+  --background-linked-wearables: #ffbc5b;
   --background-popup: #5c5c6e;
 
-  --blue: #4888EE;
-  --gray: #A19EA8;
+  --blue: #4888ee;
+  --red: #d80027;
+  --gray: #a19ea8;
+  --dark-gray: #736e7d;
+  --dark-gray-rgb: 115, 110, 125;
```

Comment:
You can include rgb/rgba here. If the color has opacity I'd add it here as a color too.
For example, we can have `dark-gray` and `dark-gray-opacity` or similar.

Reply:
Yes I know, but actually the opacity depends on the component that uses the color. That's why I defined it this way, I leave you some examples below:

![image](https://user-images.githubusercontent.com/45410089/174107900-2be10d2a-0377-4e51-a134-61436abc2cde.png)
![image](https://user-images.githubusercontent.com/45410089/174108066-a475b747-1d03-4783-8557-baf92bf13540.png)
![image](https://user-images.githubusercontent.com/45410089/174108233-9e8de90a-b3e2-4274-9dcf-dd710ca7991e.png)
![image](https://user-images.githubusercontent.com/45410089/174108552-c06f2d6c-0662-456c-b396-2511d2b1d517.png)

We could agree on a standard with Juani and define an opacity value, WDTY?

Reply:
Yes! Let's define a standard opacity value